scripts/build.js

```diff
@@ -6,7 +6,8 @@ const outputFile = path.join(__dirname, '..', 'dpdb.json');
 const formattedJSON = fs.readFileSync(inputFile, 'utf8');
 
 // Use JSON.stringify/parse to ensure that the JSON is valid
-// rather than just stripping out space/newline chars
-const minifiedJSON = JSON.stringify(JSON.parse(formattedJSON));
+// rather than just stripping out space/newline chars.
+const formatify = (str, indent=0) => JSON.stringify(JSON.parse(str), null, indent);
 
-fs.writeFileSync(outputFile, minifiedJSON);
+fs.writeFileSync(inputFile, formatify(formattedJSON, 2));
```

Comment:
I'd maybe check to see that if the file is improper JSON, it does not overwrite anything (I think it should just throw and stop, though)